src/Shared/runtime/Http2/Hpack/H2StaticTable.Http2.cs

```diff
@@ -21,7 +21,7 @@ status switch
                 400 => 12,
                 404 => 13,
                 500 => 14,
-                _ => throw new ArgumentOutOfRangeException()
```

Comment:
This will need to be reflected in the Runtime repo as well

Reply:
Done - https://github.com/dotnet/runtime/pull/54161